content/community.md

```diff
@@ -68,3 +68,9 @@ The Ratis issues mailing list is: <issues@ratis.apache.org>.
 ### Slack
 
 You can join the #ratis channel in Apache slack workspace [http://s.apache.org/slack-invite](http://s.apache.org/slack-invite) for any discussions.
+
+### JIRA Issue Tracker
+
+The JIRA issue tracker for Ratis is at [https://issues.apache.org/jira/projects/RATIS/](https://issues.apache.org/jira/projects/RATIS/) .
+If you are new to JIRA Issue Tracker,
+please see the [Introduction](https://issues.apache.org/jira/secure/Dashboard.jspa).
```

Comment:
Is the dashboard link correct for Introduction?

Reply:
There is currently an introduction in the dashboard.  Not sure if it will be changed in the future.  Anyway, let's remove the dashboard link since it is not very useful.